dev/com.ibm.ws.jca_fat/bnd.bnd

```diff
@@ -18,6 +18,11 @@ src: \
 
 fat.project: true
 
+# Define additional tested features that are NOT present any any XML files in this bucket.
```

Comment:
Minor Comment: should read "in any" ... I think the same typo got copied to the other bnd files as well.

Reply:
Will fix.  What is bad is I copied it from another place as well.

Reply:
I have made the changes.  If you are satisfied please mark the conversation resolved.